src/SDKs/SqlManagement/Management.Sql/Generated/SdkInfo_SqlManagementClient.cs

```diff
@@ -31,7 +31,10 @@ internal static partial class SdkInfo
                 new Tuple<string, string, string>("Sql", "FailoverGroups", "2015-05-01-preview"),
                 new Tuple<string, string, string>("Sql", "FirewallRules", "2014-04-01"),
                 new Tuple<string, string, string>("Sql", "GeoBackupPolicies", "2014-04-01"),
+                new Tuple<string, string, string>("Sql", "InstanceFailoverGroups", "2017-10-01-preview"),
```

Comment:
Please run `msbuild build.proj /t:build /p:Scope=SDKs\SqlManagement` and commit any artifacts that may be generated

Reply:
no new files were generated

Reply:
This file should have some changes
https://github.com/jugeorge/azure-sdk-for-net/blob/8862cab86643c1c7dadf6228ceba37c277d87332/src/SDKs/SqlManagement/AzSdk.RP.props